config/config.go

```diff
@@ -42,7 +42,7 @@ type Plan struct {
 	ID           string `yaml:"plan_id"`
 	Name         string `yaml:"plan_name"`
 	Description  string `yaml:"description"`
-	StorageClass string `yaml:"kube_storage_class"`
+	StorageClass *string `yaml:"kube_storage_class"`
```

Comment:
Hmm, Did `go fmt` not run correctly? (I'd have expected the `yaml`s to line up.)

Reply:
My go 1.13 always fails with 
```go: k8s.io/client-go@v2.0.0-alpha.0.0.20181213151034-8d9ed539ba31+incompatible: invalid pseudo-version: preceding tag (v2.0.0-alpha.0) not found
```
I managed to build using `docker build .`, which explicitly uses `go 1.12`.
Ok. managed to run indirectly by inserting `go fmt -x` into the dockerfile.
And the used `gofmt` directly, as reported by the `-x` option above.

Two files got reported, one is my changes, the other seems to have existed for longer.

Done with commit 6faa8cade3aac25f762d8ecad39423a760d71875